.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+- id: urlchecker
```

Comment:
I'm not familiar with pre commit hooks - how would this file be used, and where? Are there instructions here you are adding for how the user is supposed to interact with this file?

Reply:
I will add a section to the documentation similar to https://github.com/psf/black#version-control-integration. Where would you like it?

If you'd like to test in a test repo, you need to point to my branch temporarily in your config.

```
repos:
-   repo: https://github.com/aerickson/urlchecker-python
    rev: 067c4c5deab909718c9c3f586b0adacf70e1c078
    hooks:
    - id: urlchecker
```

Reply:
So is repos a part of GitHub actions? Something else?

Could you please set up a test repository to show me how it works, and if we integrate it here, we can point the user to it as an example?

Reply:
That blob of text is a complete .pre-commit-config.yaml file that lives in the root of a repository.

Yeah, absolutely.

Reply:
Here's my test repo. Setup/testing directions are in the README.md.

https://github.com/aerickson/test_repo

Reply:
Oh this will be hugely helpful! I'll try to review this soon, but realistically I can make a chunk of time over the weekend. Stay tuned! @SuperKogito you might also want to test this out to give feedback.